packages/cli-dxns/package.json

```diff
@@ -22,12 +22,12 @@
     "dist/"
   ],
   "scripts": {
```

Comment:
could you please update "bin" section of this package.json, since it says `dx-halo` instead of `dx-dxns`

Reply:
Done.